### ISSUE TITLE
Document Table Of Contents Slides Better

### ISSUE THREAD

**Issue by MartinPacker** [MartinPacker/md2pptx#78]
Right now only styling is really described.

The need to have the TOC slide coded up with a suitable title and the following format is needed:

```
* [Some Trends We See](#some-trends-we-see)
* [Availability Considerations](#availability-considerations)
* [Performance Considerations](#performance-considerations)
* [Other Considerations](#other-considerations)
* [Asymmetry](#asymmetry)
* [Of DB2 And MQ](#of-db2-and-mq)
* [While We're Getting Fancy With Architecture](#while-were-getting-fancy-with-architecture)
* [Conclusion](#conclusion)
```

The above links are internal so need coding appropriately in the section headings that are targeted.

**Comment by MartinPacker** [comment]
The above list of links can be generated by mdpre - so probably best to reference how mdpre does it in a discussion of TOC slides.

**Comment by MartinPacker** [comment]
Issue 94 gives me the excuse to work on this.

**Comment by MartinPacker** [comment]
Shipped in v2.3.2.
